PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -4919,7 +4919,7 @@ LRESULT FAR PASCAL FindReplaceDlg::comboEditProc(HWND hwnd, UINT message, WPARAM
 	else if ((message == WM_KEYDOWN) && (wParam == VK_DOWN) && (::SendMessage(hwndCombo, CB_GETCURSEL, 0, 0) == CB_ERR))
 	{
 		// down key on unselected combobox item -> store current edit text as draft
-		::SendMessage(hwndCombo, WM_GETTEXT, FINDREPLACE_MAXLENGTH - 1, reinterpret_cast<LPARAM>(draftString.get()));
+		::SendMessage(hwndCombo, WM_GETTEXT, FINDREPLACE_MAXLENGTH, reinterpret_cast<LPARAM>(draftString.get()));

```

Comment:
The API for WM_GETTEXT, [here](https://learn.microsoft.com/en-us/windows/win32/winmsg/wm-gettext) says that "wParam is the maximum number of characters to be copied, including the terminating null character."  Thus, we should not subtract 1 from FINDREPLACE_MAXLENGTH here; doing so limits the number of characters copied to FINDREPLACE_MAXLENGTH - 2 which is not what we want.